### DISCUSSION TITLE
I created configuration generator and docker compose for Outline

### DISCUSSION THREAD

**Discussion by chsasank** [outline/outline#1345] [General]
I have self hosted outline for our startup. I have written a blog post describing why we chose outline.
I have also created a github repo for new users to generate configuration and self host easily using docker-compose.

https://github.com/chsasank/outline-wiki-docker-compose

http://chsasank.github.io/outline-self-hosted-wiki.html

Any suggestions are welcome! Thanks!

I'm willing to help outline integrate this repo into their main docs, if they're interested.

**Comment by tommoor** [comment]
Nice nice, hopefully some of the ideas in this can be brought across to the core repo

**Reply by chsasank** [reply]
How do I help? I'm a noob for node.js apps. If you guide me through it, (like how you did in https://github.com/outline/outline/discussions/1337) I'll help integrate this into your documentation.
